### PR TITLE
Draft CYF Website

### PR DESCRIPTION
**Pull request by panditita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            </div>
+        </div>
+    </header>
+    <div class="main-container">
+        <div class="row">
+            <div class="primary-content col-lg-8">
+                <section class="about">
+                    <h1> About C*Y*F </h1>
+                    <h3> What do we do?</h3>
+                    <p><span style="font-weight:bold; font-size:16pt"> CYF </span> is a fictious company.Lorem ipsum dolor sit
```
Comment (Happy0) [inline index.html]: Maybe this inline style could be moved into a CSS class / id instead? :)

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    evt.preventDefault();
+    var nameArea = document.querySelector("#nameArea");
+    if (nameArea.value == "") {
+        alert("Error: Name is empty!");
+        nameArea.focus();
+        return false;
+    }
+}
+
+var submitBtn = document.querySelector('#submitBtn'); // Element that is going to call the action
```
Comment (Happy0) [inline js/main.js]: You already have a reference to your 'Submit Button' on line 12 as the 'submitBtn' variable.

You can just reuse this variable :)

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        x.style.display = '';
+    }
+}
+
+var submitBtn = document.querySelector('#submitBtn'); // Element that is going to call the action
+console.log(submitBtn);
+submitBtn.addEventListener("click", checkName);
+
+function checkName(evt) {
+    evt.preventDefault();
```
Comment (Happy0) [inline js/main.js]: Nice one :)

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    var nameArea = document.querySelector("#nameArea");
+    if (nameArea.value == "") {
+        alert("Error: Name is empty!");
+        nameArea.focus();
```
Comment (Happy0) [inline js/main.js:19]: Nice

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+console.log(submitBtn);
+submitBtn.addEventListener("click", checkForm);
+
+function checkMobileNumber (evt){
+    evt.preventDefault();
+    var phoneArea = document.querySelector("#phoneArea");
+    if (phoneArea.value=""){}
+}
+*/
+var submitBtn = document.querySelector('#submitBtn'); // Element that is going to call the action
```
Comment (Happy0) [inline js/main.js]: You already have a reference to your 'Submit Button' on line 12 as the 'submitBtn' variable.

You can just reuse this variable :)

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    var phoneArea = document.querySelector("#phoneArea");
+    for (var i = 0; i < phoneArea.value.length; i++) {
+        if (phoneArea.value.length != 10 && phoneArea.value.length != 11) {
+            alert("Please enter a valid UK phone number.")
+            return false
+        }
+        if (isNaN(phoneArea.value[i]) && parseInt(Number(phoneArea.value[i])) != phoneArea.value[i] &&
+            isNaN(parseInt(phoneArea.value[i], 10))) {
+            alert('Please provide a valid phone number');
+            phoneNumber.focus;
```
Comment (Happy0) [inline js/main.js]: I think you meant phoneNumber.focus() here? 

The 'focus' function won't be called without adding ().

File: styles/style.css [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+  flex: 1;
+}
+
+button.nav-item-1.btn{
+  flex: 1;
+}
+button.nav-item-2.btn{
+  flex: 1;
+}
+button.nav-item-3.btn{
```
Comment (Happy0) [inline styles/style.css]: You've got 3 CSS classes here that do the same thing. Perhaps you could just reuse one of them?

**Review by Happy0 — COMMENTED** [review]
Looks good overall :). 

I haven't looked at what the website looks like, I haven't looked at it in a web browser.

I've added some comments. Let me know if any of them are confusing.

File: styles/style.css (hunk 7 above)
Comment (panditita) [reply styles/style.css]: I know! i'm not sure how to group them

File: js/main.js (hunk 2 above)
Comment (panditita) [reply js/main.js]: Oh so I don't need to create the variable each time?!

File: js/main.js (hunk 6 above)
Comment (panditita) [reply js/main.js]: I didn't realise I was missing the () maybe that's why it wasn't working properly

File: styles/style.css (hunk 7 above)
Comment (Happy0) [reply styles/style.css]: Hmm... Those classes (nav-item-1, nav-item-2, nav-item-3) don't seem to be used anywhere, right?

You have used the 'nav-item' and 'btn' classes here: https://github.com/panditita/WebDeveloperTest/blob/master/index.html#L22

So why don't you just have one 'button.nav-item.btn' class in style.css instead? Also, I think the 'button.' part of the selector might be too specific - I could imagine this class being used on a navigation item that wasn't a button, but a div or span instead if you know what I mean? Wouldn't  'nav-item.btn' be enough?

**Pull request closed without merging**